vsintegration/tests/Salsa/salsa.fs

```diff
@@ -34,6 +35,19 @@ module internal Salsa =
 
     exception MarkerNotFoundException of string
 
+    type Async with
+        static member RunImmediate (computation: Async<'T>, ?cancellationToken ) =
```

Comment:
Was it missing from salsa suite?

Reply:
Hmmm looks like this crept in by mistake, I'll remove it